feature/experimental/tunnel/otg_tests/tunnel_loadbalance_acl_based/tun_acl_loadbalance_test.go

```diff
@@ -0,0 +1,868 @@
+	aclConf := gnmi.OC().Acl().Interface(ifName)
+	if ifName != "" {
+		iFace.GetOrCreateIngressAclSet(aclIpv4Name, oc.Acl_ACL_TYPE_ACL_IPV4)
+		iFace.GetOrCreateInterfaceRef().Interface = ygot.String(ifName)
+		iFace.GetOrCreateInterfaceRef().Subinterface = ygot.Uint32(0)
+	} else {
+		iFace.GetOrCreateIngressAclSet(aclIpv4Name, oc.Acl_ACL_TYPE_ACL_IPV4)
+		iFace.DeleteIngressAclSet(aclIpv4Name, oc.Acl_ACL_TYPE_ACL_IPV4)
+	}
+	return aclConf
+}
+func configACLIpv6Interface(t *testing.T, iFace *oc.Acl_Interface, ifName string) *acl.Acl_InterfacePath {
+	aclConf := gnmi.OC().Acl().Interface(ifName)
+	if ifName != "" {
+		iFace.GetOrCreateIngressAclSet(aclIpv6Name, oc.Acl_ACL_TYPE_ACL_IPV6)
+		iFace.GetOrCreateInterfaceRef().Interface = ygot.String(ifName)
+		iFace.GetOrCreateInterfaceRef().Subinterface = ygot.Uint32(0)
+	} else {
+		iFace.GetOrCreateIngressAclSet(aclIpv6Name, oc.Acl_ACL_TYPE_ACL_IPV6)
+		iFace.DeleteIngressAclSet(aclIpv6Name, oc.Acl_ACL_TYPE_ACL_IPV6)
+	}
+	return aclConf
+}
+
+// configureNetworkInstance Configure default network instance
+func configureNetworkInstance(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	dutConfPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut))
+	gnmi.Replace(t, dut, dutConfPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+	configureStaticRoute(t, dut, prefix, nexthop)
+	configureStaticRoute(t, dut, ipv6Prefix, ipv6Nexthop)
+}
+
+// configStaticRoute configures a static route.
+func configureStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) {
+	ni := oc.NetworkInstance{Name: ygot.String(deviations.DefaultNetworkInstance(dut))}
+	static := ni.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, deviations.StaticProtocolName(dut))
+	sr := static.GetOrCreateStatic(prefix)
+	nh := sr.GetOrCreateNextHop("0")
+	nh.NextHop = oc.UnionString(nexthop)
+	gnmi.Update(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, deviations.StaticProtocolName(dut)).Config(), static)
+}
+
+func validatePackets(t *testing.T, filename string) {
+	handle, err := pcap.OpenOffline(filename)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer handle.Close()
+	packetSource := gopacket.NewPacketSource(handle, handle.LinkType())
+	cntNonGrePackets := 0
+	ipLayerNillcount := 0
+	for packet := range packetSource.Packets() {
+		ipLayer := packet.Layer(layers.LayerTypeIPv4)
+		ethLayer := packet.Layer(layers.LayerTypeEthernet)
+		ethPacket, _ := ethLayer.(*layers.Ethernet)
+
+		if ethPacket == nil {
+			fmt.Println("EthernetType:", ethPacket.EthernetType)
+		}
+		resetcounter := 0
+		if ipLayer == nil {
+			resetcounter++
+			ipLayerNillcount++
+
+		}
+		fmt.Println("cntNillIpLayer:", resetcounter)
+		if resetcounter == 0 {
+			ipPacket, _ := ipLayer.(*layers.IPv4)
+			fmt.Printf("IpLayer is : %d ", ipPacket)
+			//	ipPacket, _ := ipLayer.(*layers.IPv4)
```

Comment:
Remove the code if not needed.

Reply:
Removed the lines.
			ipPacket, _ := ipLayer.(*layers.IPv4)
			fmt.Printf("IpLayer is : %d ", ipPacket)